src/microbit/pxt_vis_navnet_ditt/pxt_vis_navnet_ditt.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "PXT: Vis navnet ditt"
+author: Helene Isnes
+language: nb
+---
+
+
+# Introduksjon {.intro}
+I denne introduksjonsoppgaven skal vi få microbiten til å vise navnet ditt. 
+
+
+# Steg 1: Vi finner "ved start"-klossen {.activity}
+
+## Sjekkliste {.check}
+
+- [ ] Start et nytt PXT-prosjekt, for eksempel ved å gå til 
+  [makecode.microbit.org](https://makecode.microbit.org/?lang=no).
+
+- [ ] Klikk på gruppa `Basis` som ligger øvert i menyen (midt på skjermen). 
+	Finn klossen som heter `ved start`(den ligger nederst) og klikk på den.
+
+![Bilde av ved start klossen](ved_start.png)
```

Comment:
bildet er lagret som `ved_start.PNG`, det må endres til `ved_start.png` for at bildet skal vises i oppgaven. filendingene bør være lowercase.

Reply:
Mitt utklippsverktøy klarer bare å lagre bilder som PNG. Kan jeg ha filendingene som uppercase i md filen, eller burde jeg konvertere bildefilene til png uansett?

Reply:
tror det skal funke å bare gi bildene nytt navn der du endrer filtypen manuelt, og så konverteres det automagisk

Reply:
jeg prøvde det, men med en gang jeg trykker enter blir det automatisk gjort om til PNG igjen. Eneste mulighet jeg har funnet er å åpne den i paint, slette bildet fra mappen og så lagre den på nytt som png

Reply:
da anbefaler eg at du gjør det fremover også. evt bare limer skjermbildet rett inn i paint og lagrer som .png i rett mappe. tror det var slik eg brukte å gjøre det på windows tidligere ved å bruke utklippsverktøyet.